decidim-core/app/packs/src/decidim/editor.js

```diff
@@ -22,21 +22,38 @@ export default function createQuillEditor(container) {
 
   let addImage = $(container).data("editorImages");
 
-  if (toolbar === "full") {
+  /**
+   * - basic = only basic controls without titles
+   * - content = basic + headings
+   * - multimedia = basic + image + video
+   * - full = basic + headings + image + video
+   */
+  if (toolbar === "basic") {
+    // nothing to do
+  } else if (toolbar === "content") {
     quillToolbar = [
       [{ header: [2, 3, 4, 5, 6, false] }],
+      ...quillToolbar
+    ];
+  } else if (toolbar === "multimedia") {
+    quillToolbar = [
       ...quillToolbar,
       ["video"]
     ];
-  } else if (toolbar === "basic") {
+
+    if (addImage) {
```

Comment:
The `addImage` option (through `data-editor-images`) is useless since #10011 because it is always true.

I would remove the `addImage` variable and the `data-editor-images` option completely. And rather just add the images same way as the video control for the `multimedia` and `full` toolbar types.

Also note that you need to remove it from here:
https://github.com/decidim/decidim/blob/ce5610207d4cd0763694521739b7d74d4bc0b618/decidim-core/lib/decidim/form_builder.rb#L203

The original reason for #10011 I believe was that we could copy-paste images to the rich text editor at the participant side and they became base64 encoded which we wanted to prevent. But we need to find a better way to solve that issue. If we remove the `addImage` attribute, the bug that was fixed at #10011 will reappear. But it is also kind of broken right now because it still sends the image upload request to the server. Note that the bug explained at #10011 appears only in Firefox.